public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="../src/index.css">
```

Comment:
Не корректно указан путь
Стили должны находиться рядом с index.html

Должно получиться примерно так:
```suggestion
    <link rel="stylesheet" href="%PUBLIC_URL%/index.css">
```

Это относится и к остальным местам, а также к картинкам